src/app/view/components/_common/iframe/iframe-porlet-legacy/iframe-porlet-legacy.component.ts

```diff
@@ -100,14 +100,27 @@ export class IframePortletLegacyComponent implements OnInit {
     }
 
     private setIframeSrc(): void {
-        this.route.params
-            .pluck('id')
-            .map((id: string) => id)
+        // We use the query param to load a page in edit mode in the iframe
+        const queryUrl$ = this.route.queryParams.pluck('url').map((url: string) => url);
+
+        queryUrl$.subscribe((queryUrl: string) => {
```

Comment:
If there is a queryparam in the url we set it right away, no questions asked (at least for now)